tsconfig.json

```diff
@@ -41,7 +41,7 @@
   "include": [
     "src/**/*.ts",
     "src/**/*.d.ts",
-    "src/**/*.vue"
+    "*.cjs"
```

Comment:
unrelated change - I changed our config files to be common js to appease the linter, and also removed the config for vue files since this project does not have any.